src/example_posterior_parameter_sets.txt

```diff
@@ -0,0 +1,101 @@
+iterID,nsse_cases,nsse_deaths,p_inf,p_hcw,c_hcw,d,q,p_s,rrd,intro,weight
+2, 0.122512, 0.125961, 0.153532, 0.60916, 37.9059, 0.525139, 0.313957, 0.787278, 0.516736, 8.50135e-07, 1
```

Comment:
The last column is the particle weights from the inference framework: they aren't needed for the prediction process. They can be removed.

Same for the `nsse_cases` and `nsse_deaths` columns.